pyabc/sampler/base.py

```diff
@@ -190,6 +197,7 @@ def sample_until_n_accepted(
             self,
             n: int,
             simulate_one: Callable,
+            max_eval: int = np.inf,
```

Comment:
is np.inf a type int ?

Reply:
it's the default value, i.e. effectively no limit.

Reply:
yeah, I see your point, ok. it is strictly speaking a float, not an int, but i think that won't matter here...